src/dune_site_mock/setup_paths.ml

```diff
@@ -0,0 +1,9 @@
+(*
+This file is used as a mock for the Setup_paths module generated by dune-site during the dune-based build.
+In the make-based setup we will retain the approach to find the backend proovers via the PATH environment variable.
+Hence this file inicates (by returning empty list) the absense of the dune-site-based paths, thus a fallback to PATH.
```

Comment:
```suggestion
In the make-based setup we will retain the approach of finding the backend provers via the PATH environment variable.
Hence this file indicates (by returning empty list) the absense of the dune-site-based paths, thus a fallback to PATH.
```

Reply:
The entire file is now removed. It was needed only to support two build systems in parallel.